client/src/app/site/login/components/login-mask/login-mask.component.ts

```diff
@@ -173,7 +173,7 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
             this.loginForm.setErrors({
                 notFound: true
             });
-            this.loginErrorMsg = e;
+            this.matSnackBar.open(e, 'Ok');
```

Comment:
this is weird. why do you open the snackbar? In OS3 no such snackbar opens, rather the error is shown below the password field.

Reply:
Well, it is strange, that there is no error message shown. The code is still there...
That is, why I open a snackbar...

Reply:
Please find out, why nothing is shown there

Reply:
Done.